src/main/java/frc/robot/subsystems/vision/LimelightLogInputs.java

```diff
@@ -0,0 +1,37 @@
+package frc.robot.subsystems.vision;
+
+import org.littletonrobotics.junction.AutoLog;
+import org.littletonrobotics.junction.LogTable;
+import org.littletonrobotics.junction.inputs.LoggableInputs;
+
+//@AutoLog
+//public class LimelightLogInputs {
+//    public double targetDistance = 0;
+//    public double yaw = 0;
+//    public boolean hasTargets = false;
+//    public float tagId = 0;
+//}
```

Comment:
Don't push commented-out code. If you think you'll need it in the future, just commit it and then delete it — Git will save this code in the code history.